scripts/export-results.js

```diff
@@ -0,0 +1,51 @@
+const admin = require("firebase-admin")
+const fs = require("fs")
+require("dotenv").config({
+  path: `.env.development`,
+})
+
+const getFirebase = () => {
+  const firebasePrivateKey = process.env.FIREBASE_PRIVATE_KEY
+  if (!admin.apps.length) {
+    admin.initializeApp({
+      credential: admin.credential.cert({
+        projectId: process.env.GATSBY_FIREBASE_PROJECT_ID,
+        clientEmail: process.env.FIREBASE_CLIENT_EMAIL,
+        // https://stackoverflow.com/a/41044630/1332513
+        privateKey: firebasePrivateKey?.replace(/\\n/g, "\n"),
+      }),
+    })
+  }
+  return admin
+}
+
+function getFileName() {
+  const now = new Date()
+  return `./scripts/${now.getFullYear()}-${
+    now.getMonth() + 1
+  }-${now.getDate()}----${now.getUTCHours()}H-${now.getMinutes()}min.json`
+}
+
+function writeToFile(filename, data) {
+  fs.writeFile(filename, JSON.stringify(data), err => {
```

Comment:
`JSON.stringify(data, null, 40)`  beautiful output

Reply:
I dont think we need to make it beautiful as it would be larger and we only need to use the output for playground not for user to understand